backend/server.js

```diff
@@ -109,3 +110,82 @@ app.post("/player", async (req, res) => {
   });
   res.json(newPlayer);
 });
+
+app.post("/myTeamPlayer", async (req, res) => {
+  const { performanceScore, playerId } = req.body;
+
+  const newPlayer = await prisma.myTeamPlayer.create({
+    data: {
+      performanceScore,
+      playerId,
+    },
+  });
+  res.json(newPlayer);
+});
```

Comment:
While it's tempting to add all of these api calls now - I think it might make sense to instead add them at the same time you need to use them. So for example, you'd add this post call in the same pull request as when you need to add to your db! That way you'll keep your code lean and dont add extra code that might not be needed